View/ImpactAreaScenario/Results/DamageByDamageCategory.xaml

```diff
@@ -26,7 +26,7 @@
                         <TextBlock Text="Damage Category" TextWrapping="Wrap" />
                     </DataGridTextColumn.Header>
                 </DataGridTextColumn>
-                <DataGridTextColumn Binding="{Binding EAD}" Width="*">
+                <DataGridTextColumn Binding="{Binding EAD, StringFormat=N2}" Width="*">
```

Comment:
I've encountered this a few times in your new code, I think we should consider doing a static string for this somewhere like a class called UnitFormatUtils.  This could be changed to StringFormat={x:Static namespace:UnitFormatUtils.DEFAULT} or something of that nature.  it'll let you change it in one place and be consistent across the board if they want to adjust it to one or two or five or whatever.